docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md

```diff
@@ -122,7 +122,7 @@ In this step, you set up a type provider with a data connection and obtain a dat
 type private EntityConnection = SqlEntityConnection<ConnectionString="Server=SERVER\InstanceName;Initial Catalog=School;Integrated Security=SSPI;MultipleActiveResultSets=true",Pluralize = true>
 ```
 
-  This action sets up a type provider with the database connection that you created earlier. The property `MultipleActiveResultSets` is needed when you use the ADO.NET Entity Framework because this property allows multiple commands to execute asynchronously on the database in one connection, which can occur frequently in ADO.NET Entity Framework code. For more information, see [Multiple Active Result Sets (MARS)](http://go.microsoft.com/fwlink/?LinkId=236929).
+  This action sets up a type provider with the database connection that you created earlier. The property `MultipleActiveResultSets` is needed when you use the ADO.NET Entity Framework because this property allows multiple commands to execute asynchronously on the database in one connection, which can occur frequently in ADO.NET Entity Framework code. For more information, see [Multiple Active Result Sets (MARS)](https://docs.microsoft.com/en-us/sql/relational-databases/native-client/features/using-multiple-active-result-sets-mars).
```

Comment:
instead of https://docs.microsoft.com/en-us/sql/relational-databases/native-client/features/using-multiple-active-result-sets-mars, the link should look like /sql/relational-databases/native-client/features/using-multiple-active-result-sets-mars

This will ensure that it works for offline books and other locales too.

Reply:
Okay!, What is the pattern I should look for which should look like this?

Reply:
@xaviex for links going to docs.microsoft.com/en-us/xyz should go to /xyz. Except when the xyz is dotnet. Then the preferred way would be ~/docs/<path-after-dotnet>.md

For example, https://docs.microsoft.com/en-us/dotnet/core/docker/ would become ~/docs/core/docker.md

Reply:
got it, Implementing this now for future links